changelogs/fragments/4479-add-project-support-for-lxd_container-and-lxd_profile.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - lxd_container - adds ``project`` option to allow selecting project for lxd instance (https://github.com/ansible-collections/community.general/pull/4479).
+  - lxd_profile - adds ``project`` option to allow selecting project for lxd profile (https://github.com/ansible-collections/community.general/pull/4479).
```

Isn't the official spelling `LXD` (https://linuxcontainers.org/)?
```suggestion
  - lxd_container - adds ``project`` option to allow selecting project for LXD instance (https://github.com/ansible-collections/community.general/pull/4479).
  - lxd_profile - adds ``project`` option to allow selecting project for LXD profile (https://github.com/ansible-collections/community.general/pull/4479).
```

---

Thanks, fixed the spelling